public/fonts/satoshi.css

```diff
@@ -0,0 +1,178 @@
+/**
+ * @license
+ *
+ * Font Family: Satoshi
+ * Designed by: Deni Anggara
+ * URL: https://www.fontshare.com/fonts/satoshi
+ * © 2022 Indian Type Foundry
```

Comment:
Still have some concerns about this…

<img width="517" alt="image" src="https://user-images.githubusercontent.com/13428215/167055733-fd3a4597-1bdd-4219-9655-5e2bc9ff2a1a.png">

Reply:
https://www.fontshare.com/terms#limitations-of-usage

Reply:
we can use the fonts through the FontShare API which realized was available so we can delete the font files on our end altogether, so we're not sharing the font files

Reply:
Deleted 'font' folder.
Imported API for Font in style.css